library/api/google_street_view/GoogleStreetView_3.7.js

```diff
@@ -1,222 +0,0 @@
-lizMap.events.on({
```

Comment:
Please do not remove the 3.7 file.

LWC 3.7 is still maintained and used, we want to keep the script for this version.

It will be like https://github.com/3liz/lizmap-javascript-scripts/tree/master/library/misc/external_links_from_map_click for instance.